cmake/External/nnpack.cmake

```diff
@@ -51,6 +51,13 @@ if(ANDROID OR IOS OR ${CMAKE_SYSTEM_NAME} STREQUAL "Linux" OR ${CMAKE_SYSTEM_NAM
   if(NOT DEFINED CPUINFO_SOURCE_DIR)
     set(CPUINFO_SOURCE_DIR "${CAFFE2_THIRD_PARTY_ROOT}/cpuinfo" CACHE STRING "cpuinfo source directory")
   endif()
+  # Orlando: check if avx bytecode has been generated or not
+  # clean up egg-link
+  execute_process(COMMAND python "${PYTHON_PEACHPY_SOURCE_DIR}/setup.py" develop --uninstall
+      WORKING_DIRECTORY "${PYTHON_PEACHPY_SOURCE_DIR}")
+  # reinstall and overwrite if instructions are generated
+  execute_process(COMMAND python "${PYTHON_PEACHPY_SOURCE_DIR}/setup.py" develop
+          WORKING_DIRECTORY "${PYTHON_PEACHPY_SOURCE_DIR}")
```

Comment:
I'm a bit at loss here, can you please explain what problem is change is fixing? Does it fix the build between peachpy updates? Or is it always happen even on the clean installs?

Reply:
@malfet currently, when building pytorch, peachpy haven't been triggered to generate instructions.  so when doing the building, we will meet with the issue, refer to https://github.com/pytorch/pytorch/issues/9257. the missing step is to generate instructions, so peachpy.x86_64.avx can't be found under the source folder. when peachpy has been updates, "python setup.py develop --uninstall" will clean up the installation information, "python setup.py develop" will install new peachpy version. If any instructions has been generated by previous version, they can be overwritten by the latter commands. 

And now as peachpy has been only installed by sources under third_party. Even if we use pip install, it couldn't work. That is why I updated cmake scripts for nnpack.

Reply:
@llv22 are you saying that `peachpy` is installed on your system?
Can you please run `python -c 'import peachpy;print(peachpy.__file__)'` and share the output here?

If the problem stems from the fact that peachpy installed in the system path conflicts with peachpy from third_party folder then we should modify script that uses `PEACHPY_PYTHONPATH`

Reply:
@malfet actually I mean that 

- in pytorch building, it **actually refer to peachpy under third_party folder**: this is right, because this can guarantee latest version of peachpy is used
-  in cmake script, we overwrite PYTHON_PATH for peachpy when triggering pytorch building (I will give out this script name later, needed to double-check). Therefore, even if you have another version of peachpy installed in system, there would be no conflicts between peachpy used for pytorch building and peachpy installed in system(in fact my mac is exactly under this case)

What I did in the PR is to guarantee that peachpy source used by pytorch building can generate instruction codes. And this step is missing in nnpack's cmake script.